components/common-util/api/index.tsx

```diff
@@ -0,0 +1,49 @@
+import qs from 'qs';
+
+const URL = `${process.env.NEXT_PUBLIC_API_URL}/api`;
```

Comment:
if there are env variables used in repo, it's a good idea to help engineers who see the project for the first time to add .env.example (like in olas-website) and list all possible variables so that they can be easily added before first run